libs/remix-ui/solidity-compiler/src/lib/logic/compileTabLogic.ts

```diff
@@ -133,6 +139,12 @@ export class CompileTabLogic {
             this.api.logToTerminal({ type: 'error', value: error })
           })
         }
+      } else if (externalCompType === 'truffle') {
```

Comment:
`this.api.getFileManagerMode() === 'localhost'` should be in the check too.

Reply:
ah yes

Reply:
updated